libraries/AP_Scripting/applets/SM_SwitchMission.lua

```diff
@@ -0,0 +1,320 @@
+--[[
+
+SM_SwitchMission.lua - ArduPilot Lua script
+
+That script is derived from MissionSelector.lua in  
+https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_Scripting/applets  
+and the examples in https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_Scripting/examples  
+  
+This script is useful to switch between different missions on SD-card vis RC-channel
+
+The switching can be done by multi-position-switch or pushbutton.
+The switching is allowed: 
+  -disarmed, 
+  -in modes that are not mode-AUTO if they are activated by RC or GCS
+  -in mode-AUTO when the current mission is complete
+
+CAUTION: Use this script AT YOUR OWN RISK.
+
+-- Willy Zehnder -- 05.12.2022
+
+]]
+-------- SCRIPT 'CONSTANTS' --------
+local SCRIPT_NAME           = 'SM'          -- abbreviation of scriptname for messages on GCS
+local SUB_DIR_APM           = '/APM/scripts'-- path to scripts on SD-card
+local SUB_DIR_SITL          = './scripts'   -- path to scripts when using SITL
+local SUB_DIR_MISSIONS      = 'missions'    -- subdirectory below scripts-directory where the mission-files are stored
```

Comment:
Perhaps "missions" should be at the APM root, rather than buried under "scripts"?